pkg/cmd/roachtest/tests/awsdms.go

```diff
@@ -68,13 +76,20 @@ const tableRules = `{
 	]
 }`
 
+var dmsEndpoints = make(map[string]string)
 var dmsTasks = []dmsTask{
-	{"test_table", tableRules, nil},
+	{"test_table", tableRules, nil, dmstypes.MigrationTypeValueFullLoadAndCdc, "default_source", "default_target"},
```

Comment:
can you add field names to all these, e.g. `tableName: "test_table", tableRules: ....`